Number_types/include/CGAL/Interval_nt.h

```diff
@@ -662,22 +926,47 @@ operator/ (const Interval_nt<Protected> &a, const Interval_nt<Protected> & b)
     return IA::largest();
 	   // We could do slightly better -> [0;infinity] when b.sup()==0,
 	   // but is this worth ?
+#endif
 }
 
 template <bool Protected>
 inline
 Interval_nt<Protected>
 operator/ (double a, const Interval_nt<Protected> & b)
 {
-  return Interval_nt<Protected>(a)/b;
+  int i = _mm_movemask_pd(_mm_cmpge_pd(b.simd(), _mm_set1_pd(0.)));
+  if(i==3) return Interval_nt<Protected>::largest(); // bi<=0 && bs>=0
```

Comment:
If seems you forgot to test `CGAL_USE_SSE2` in this function.

Reply:
Oops, yes. I actually wanted to disable those 2 overloads when we don't have SSE2, but that requires moving `operator/` to a friend first.